src/components/RecipeList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Recipe from './Recipe';
+import styled from 'styled-components';
+
+export default function RecipeList({ recipes }) {
+  return (
+    <StyledList>
+      {recipes.map(recipe => {
+        return (
+          <li key={recipe.id}>
+            <Recipe
+              title={recipe.title}
+              image={recipe.image}
+              text={recipe.summary}
+              url={recipe.sourceUrl}
+            />
+          </li>
+        );
+      })}
```

Comment:
Maybe you can try to use the map method without a return statement, if you want :)